packages/overlay/src/integrations/sentry/components/traces/spans/SpanItem.tsx

```diff
@@ -44,12 +46,16 @@ const SpanItem = ({
       setSpanNodeWidth(newLeftWidth);
     }
   };
+  const isQueried = query
+    ? span.op?.includes(query) || span.description?.includes(query) || span.span_id.includes(query)
+    : false;
```

Comment:
I'd put the check for description last as that'd be the most expensive check I presume (due to length?)

```suggestion
  const isQueried = query
    ? span.span_id.includes(query) || span.op?.includes(query) || span.description?.includes(query)
    : false;
```

Also not a fan of those `op?` and `description?` optionals but I'm guessing we have to live with it?

Reply:
Yes, it can be possible of having spans here without op or description. so we need a check here.

Reply:
Definitely i'll make the change suggested above